### PR TITLE
fix: Fixes Opacity in Figma

### PR DESCRIPTION
**Pull request by takashimokobe**
## Summary

Figma uses the values 0-100 for opacity and CSS uses 0-1. We need to update the config here so our opacity values look correct in Figma. We also need to add transformer logic in our studio sync (see [PR 100](https://github.com/Workday/canvas-tokens/pull/100).

### CODE REVIEW

File: tokens/base.json [hunk 1]
```diff
@@ -705,23 +705,23 @@
...
       "type": "number"
     },
     "200": {
-      "value": "0.12",
+      "value": "12",
       "type": "number"
     },
     "300": {
-      "value": "0.4",
+      "value": "4",
```
Comment (RayRedGoose) [inline tokens/base.json:716]: Should be this `40`?

Comment (takashimokobe) [reply tokens/base.json:716]: ah yes

File: tokens/sys/color/color.json [hunk 2]
```diff
@@ -12,12 +12,12 @@
         "description": "Transparent background"
       },
       "overlay": {
-        "value": "rgba({color.static.black},{opacity.400})",
+        "value": "rgba({color.static.black},.{opacity.400})",
```
Comment (RayRedGoose) [inline tokens/sys/color/color.json:15]: All this changes should be in different PR

**Comment by alanbsmith** [comment]
Closing. This was resolved with #21.
